packages/react-conformance/src/defaultTests.tsx

```diff
@@ -0,0 +1,69 @@
+import { TestObject, TestingOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+import { mount } from 'enzyme';
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as path from 'path';
+
+export const defaultTests: TestObject = {
+  /** Component has a docblock with 5 to 25 words */
+  'has-docblock': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
+    const maxWords = 25;
+    const minWords = 5;
+
+    // No need to check if the description is undefined, ComponentDoc.description is a "string" not "string | undefined"
```

Comment:
it would be nice to see an example using the conformance test to know how it looks.

Reply:
(not specifically here, just adding a comment)

Reply:
I'm currently working on the example!